newrelic-integration-e2e/internal/runtime/runner.go

```diff
@@ -112,7 +113,16 @@ func (r *Runner) executeOSCommands(statements []string, scenarioTag string) erro
 		cmd.Env = os.Environ()
 		cmd.Env = append(cmd.Env, "SCENARIO_TAG="+scenarioTag)
 		combinedOutput, err := cmd.CombinedOutput()
-		r.logger.Debugf("stdout: %q", combinedOutput)
+
+		if r.spec.PlainLogs {
+			r.logger.WithField("command", stmt).Info(combinedOutput)
+		} else {
+			logWriter := r.logger.Writer()
+			_, _ = fmt.Fprintf(logWriter, "::group::%s\n", stmt)
+			_, _ = fmt.Fprint(logWriter, combinedOutput)
```

Comment:
something looks odd from the logs here https://github.com/newrelic/newrelic-integration-e2e-action/runs/5023280456?check_suite_focus=true#step:4:195

Reply:
didn't find the group logging there , did you seen it in any of the self tests just to see an example?

Reply:
Yep, this approach was broken and did not work. I though that `Writer()` would return me the underlying writer, but it turns out it does not. Since it was prefixing GHA commands with all the logrus decoration, they were not being recognized.

Reply:
Groups should be working now: https://github.com/newrelic/newrelic-integration-e2e-action/runs/5024445686?check_suite_focus=true

For kafka I still see a lot of ungrouped clutter: https://github.com/newrelic/newrelic-integration-e2e-action/runs/5024445864?check_suite_focus=true

I presume these come from a different part, not the command runner. I'll take a look to see if these could be somehow grouped as well, although I'm not positive.

Reply:
Ok, I see the problem now: We are often running commands like `docker-compose -d`, which fork to the background. In these cases, the command itself exits quickly (so I close the log group) but the forked command is still writing logs to the Writer. I certainly see no way of working around this :(

Reply:
Traced the logs back to the `dockercompose` package, which is feeding stdout/stderr to the `docker-compose` process. I think we should be able to fit the CommandLogger in there as well, but unfortunately the package is just a collection of functions so I'll need to refactor it.

Reply:
I saw the folded logs , awesome !